docs/gettingstarted/install-landscaper-controller.md

```diff
@@ -16,9 +18,8 @@ helm install -n ls-system landscaper ./charts/landscaper
 We are also building OCI charts so a specific version of Landscaper can be installed with:
 
 ```
-export HELM_EXPERIMENTAL_OCI=1
 export LS_VERSION="v0.13.0" # use the latest available version
-helm chart pull eu.gcr.io/gardener-project/landscaper/charts/landscaper-controller:$LS_VERSION
+helm pull eu.gcr.io/gardener-project/landscaper/charts/landscaper-controller:$LS_VERSION
```

Comment:
```suggestion
helm pull oci://eu.gcr.io/gardener-project/landscaper/charts/landscaper --version $LS_VERSION
```

Since the format is incompatible, should we specify which is the earliest version this can be used with?